python/rrc_simulation/sim_finger.py

```diff
@@ -294,12 +294,27 @@ def _get_latest_observation(self):
             # self.__applied_torque does not exist), set it to zero
             observation.torque = np.zeros(len(observation.velocity))
 
-        finger_tip_states = pybullet.getJointStates(
```

Comment:
When this is not used anymore, the code starting in line 540 can probably be removed (unless this is also needed to get contact forces).

Reply:
okay :+1: